requirements.txt

```diff
@@ -9,14 +9,32 @@ python-dateutil==2.5.3
 strict-rfc3339==0.7
 django-cookie-law==1.0.8
 titlecase==0.8.1
+django-super-favicon==0.6.0
 ## The following requirements were added by pip freeze:
 contextlib2==0.5.3
+coverage==4.1
```

Comment:
Hmm. I just ran update-requirements.sh --new-only and committed the result. Then again, I think that script relies on virtualenv and I'm running pyvenv, which may mess with things.